spec/lcm/integration/shared_contexts_for_lcm_specs.rb

```diff
@@ -0,0 +1,264 @@
+require_relative 'support/configuration_helper'
+require_relative 'support/project_helper'
+require_relative 'support/connection_helper'
+require_relative 'support/lcm_helper'
+require 'gooddata_datawarehouse'
+
+def create_suffix
```

Comment:
Similar code found in shared_contexts_for_lcm_specs.rb:7, configuration_helper.rb:7, lcm_helper.rb:16 (mass = 90)